### ISSUE TITLE
Issue with node-sass module

### ISSUE THREAD

**Issue by dapriett** [jxcore/jxcore#492]
I have a sails app which I'm using the [node-sass](https://github.com/sass/node-sass) module.  I'm getting the below error when trying to start my app with `jx`:

```
dyld: lazy symbol binding failed: Symbol not found: __ZN4node6Buffer3NewEPKcm
  Referenced from: /sails-app/node_modules/node-sass/vendor/darwin-x64-11/binding.node
  Expected in: flat namespace

dyld: Symbol not found: __ZN4node6Buffer3NewEPKcm
  Referenced from: /sails-app/node_modules/node-sass/vendor/darwin-x64-11/binding.node
  Expected in: flat namespace

Trace/BPT trap: 5
```

Any clue what the issue could be?

**Comment by ktrzeciaknubisa** [comment]
This problem keeps arising from time to time. Please see https://github.com/jxcore/jxcore-addon/issues/1 if that helps you?

**Comment by dapriett** [comment]
Hmm - so you think it's a V8 vs SM issue?  I just install jx per directions here (https://github.com/jxcore/jxcore/blob/master/doc/INSTALLATION.md), using `curl http://jxcore.com/xil.sh | bash`

Looks like that uses V8 by default, so you think I need SM?

**Comment by dapriett** [comment]
Removed my node_modules and tries `jx install` again - noticed the following message when installing node-sass:

```
> node-sass@3.2.0 install /Users/dprietti/AndroidStudioProjects/hydra-respond/node_modules/node-sass
> /usr/local/bin/jx scripts/install.js

Binary downloaded and installed at /Users/dprietti/AndroidStudioProjects/hydra-respond/node_modules/node-sass/vendor/darwin-x64-11/binding.node
 
> node-sass@3.2.0 postinstall /Users/dprietti/AndroidStudioProjects/hydra-respond/node_modules/node-sass
> /usr/local/bin/jx scripts/build.js

` /Users/dprietti/AndroidStudioProjects/hydra-respond/node_modules/node-sass/vendor/darwin-x64-11/binding.node ` exists. 
 testing binary.
dyld: lazy symbol binding failed: Symbol not found: __ZN4node6Buffer3NewEPKcm
  Referenced from: /Users/dprietti/AndroidStudioProjects/hydra-respond/node_modules/node-sass/vendor/darwin-x64-11/binding.node
  Expected in: flat namespace

dyld: Symbol not found: __ZN4node6Buffer3NewEPKcm
  Referenced from: /Users/dprietti/AndroidStudioProjects/hydra-respond/node_modules/node-sass/vendor/darwin-x64-11/binding.node
  Expected in: flat namespace
```

**Comment by ktrzeciaknubisa** [comment]
> Looks like that uses V8 by default, so you think I need SM?

Nope. I meant that if you use one binary for `jx install` and you have another one in the path, that may be the problem :) 

And I see the same error when `jx install node-sass`, too.

**Comment by dapriett** [comment]
Ah, ok - I just have the v8 engine installed for now.

**Comment by ktrzeciaknubisa** [comment]
And is this solving anything? For me not. The part from `jx install output`:

>  Binary downloaded and installed at /Users/dprietti/AndroidStudioProjects/hydra-respond/node_modules/node-sass/vendor/darwin-x64-11/binding.node

suggests that the the *binding.node* was downloaded instead of compiled, thus may not work with JXcore if it was precompiled against node. Am I right @obastemur ?

**Comment by dapriett** [comment]
Looks like their install script just downloads from here based on platform and arch: https://github.com/sass/node-sass/releases

**Comment by obastemur** [comment]
> suggests that the the binding.node was downloaded instead of compiled, thus may not work with JXcore if it was precompiled against node. Am I right @obastemur ?

Right. Don't know if it accepts any force-compile switch.

**Comment by dapriett** [comment]
Dug into their source, looks like you can do `SASS_FORCE_BUILD=true jx install node-sass`.  Try it though, but still get the same error when running my app....

**Comment by ktrzeciaknubisa** [comment]
Well, it forces the build indeed, but apparently not of `/node_modules/node-sass/vendor/darwin-x64-11/binding.node` (still there is a *Binary downloaded and installed* message...)

**Comment by ktrzeciaknubisa** [comment]
This issue is fixed with new npm 3.3.12 which comes with JXcore 0.3.1.1
